examples/tracing-jaeger/src/main.rs

```diff
@@ -29,7 +29,8 @@ fn init_tracer() -> Result<opentelemetry_sdk::trace::Tracer, TraceError> {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
-    let _tracer = init_tracer().expect("Failed to initialize tracer.");
+    let tracer_provider = init_tracer().expect("Failed to initialize tracer provider.");
+    global::set_tracer_provider(tracer_provider.clone());
```

Comment:
the code at the end of this file uses the global::shutdown_tracer_provider. Lets fix that as well.
i.e modify the example to call shutdown on the above tracer_provider, but don't remove the existing global::shutdown_tracer_provider public api for now. (that need separate discussion)

Reply:
and ofcourse, do this for all the examples! 
Ideally - please manually validate the examples still work by verifying traces in collector!